ci/upgrade.sc

```diff
@@ -107,10 +181,45 @@ def updateBuildInfo(url: String, sha1: String, repoPath: Path, fileName: String,
   )
 
   // Save again.
-  val prettyJson = Json.prettyPrint(Json.toJson(updatedBuildInfo))
+  val prettyJson = prettyPrint(Json.toJson(updatedBuildInfo))
   write.over(buildInfoPath, s"$prettyJson\n")
 }
 
+/**
```

Comment:
Is this just so we can write the buildinfo.json in DC/OS ? Maybe we should just use a string template?

Reply:
hmm ... I think you're right. That sounds like better approach :) Let me iterate on that

Reply:
no actually I thought about it and I can't because for EE I need to pull in the changes from master - I want to use the last plugins etc. and just alter the metronome/marathon version. I don't want to hardcode plugins version into the template

Reply:
Not sure I follow, but at the end, you could just make it separate PRs ;)

Reply:
yeah we can do that, but then if we merge those two again in the future (and I hope we do) we will have to solve this again :/

Reply:
Even if we separate, I would like to merge this in still so still looking for your opinions @zen-dog @timcharper